control/trajectory_follower/design/mpc_lateral_controller-design.md

```diff
@@ -22,29 +22,64 @@ The node uses an implementation of linear model predictive control (MPC) for acc
 The MPC uses a model of the vehicle to simulate the trajectory resulting from the control command.
 The optimization of the control command is formulated as a Quadratic Program (QP).
 
-These functionalities are implemented in the `trajectory_follower` package
-(see [trajectory_follower-mpc-design](../../trajectory_follower/design/trajectory_follower-mpc-design.md#mpc-trajectory-follower))
+Different vehicle models are implemented:
 
-### Assumptions / Known limits
+- kinematics : bicycle kinematics model with steering 1st-order delay.
+- kinematics_no_delay : bicycle kinematics model without steering delay.
+- dynamics : bicycle dynamics model considering slip angle.
+  The kinematics model is being used by default. Please see the reference [1] for more details.
+
+For the optimization, a Quadratic Programming (QP) solver is used with two options are currently implemented:
```

Comment:
```suggestion
For the optimization, a Quadratic Programming (QP) solver is used and two options are currently implemented:
```

Reply:
corrected in https://github.com/autowarefoundation/autoware.universe/pull/1591/commits/ebeed6c2e8576ce9e879132a04ac63294e815c65